drivers/flash/nrf_qspi_nor.c

```diff
@@ -1262,51 +1263,51 @@ static const struct flash_driver_api qspi_nor_api = {
 #endif /* CONFIG_FLASH_JESD216_API */
 };
 
-#ifdef CONFIG_PM_DEVICE
-static int enter_dpd(const struct device *const dev)
+static int exit_dpd(const struct device *const dev)
```

Comment:
Please don't move this function. This produces much bigger diff and unnecessarily complicates checking history for both `enter_dpd()` and `exit_dpd()`.
Instead, you can do just:
```diff
 }
+#endif /* CONFIG_PM_DEVICE */
+
 static int exit_dpd(const struct device *const dev)
```
```diff
 }
+
+#ifdef CONFIG_PM_DEVICE
 static int qspi_nor_pm_action(const struct device *dev,
```

Reply:
@anangl I thought it might be better to group the functionality behind the `CONFIG_PM_DEVICE` flag together, but I agree it complicates the diff.
I made the change per your request, thanks for your feedback!